sdcm/cluster.py

```diff
@@ -4083,10 +4085,31 @@ def stop_nemesis(self, timeout=10):
             return
         self.log.info('Set _nemesis_termination_event')
         self.nemesis_termination_event.set()
+        threads_tracebacks = []
+        # pylint: disable=protected-access
+        current_thread_frames = sys._current_frames()
         for nemesis_thread in self.nemesis_threads:
             nemesis_thread.join(timeout)
+            if nemesis_thread.is_alive():
+                stack_trace = traceback.format_stack(current_thread_frames[nemesis_thread.ident])
+                threads_tracebacks.append("\n".join(stack_trace))
+        self.argus_terminate_remaining_nemeses(stack_trace="\n".join(threads_tracebacks))
         self.nemesis_threads = []
 
+    def argus_terminate_remaining_nemeses(self, stack_trace="") -> None:
+        try:
+            self.log.info("Attempting to terminate remaining nemeses.")
```

Comment:
log number of Nemesis running, something like 
`"Attempting to terminate 3 remaining nemeses."`